src/backend/joanie/core/models/products.py

```diff
@@ -1140,18 +1140,25 @@ def set_installment_refused(self, installment_id):
         ActivityLog.create_payment_failed_activity_log(self)
         self._set_installment_state(installment_id, enums.PAYMENT_STATE_REFUSED)
 
+    def get_installments(self, state, get_indexes=False):
```

Comment:
I don't really like the idea to have two different behavior for a single function. Create 2 dedicated function is a better idea IMO